### PR TITLE
Updates to email heatmaps and email settings layout

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/administrative/app_settings/manage_app_group/email_settings.md [hunk 1]
```diff
@@ -11,94 +11,65 @@ channel: email
...
+The email settings under the **Sending Configuration** section determine which details are included in your email campaigns. In particular, these settings are mainly related to what your user sees when they receive an email from Braze.
 
-Use the following outbound email settings to change the name and email address used when Braze sends emails to your users. Using these settings, you may also include additional sending and reply-to addresses. These addresses will be available when building your campaign through the **Edit Sending Info** options available as you compose your campaign email.
+### Outbound email settings
 
-Here, you may also exclude the "Reply-To" email and exclusively send replies to the "From" address.
+When configuring your email settings, your outbound email settings identify which name and email addresses are used when Braze sends emails to your users.
 
-![Outbound Email Settings section of the Email Settings tab.][1]
+{% tabs %}
```
Comment (KellieHawks) [inline _docs/_user_guide/administrative/app_settings/manage_app_group/email_settings.md:24]:
```suggestion
{% tabs local %}
```

File: _docs/_user_guide/administrative/app_settings/manage_app_group/email_settings.md [hunk 2]
```diff
@@ -110,74 +81,92 @@ Once saved, Braze will send special instructions to the ESP in order to place th
...
-When toggled on, this will append the word "[TEST]" for emails sent as a test, or "[SEED]" when emails are send to your users. By default, this feature is disabled.
-  
-![Test and Seed Email Subject Lines section in the Email Settings page.][14]
+You may automatically resubscribe users when they change their email address. For example, if a previously unsubscribed app group user changes their email address to one that is not on Braze's unsubscribe list, they will automatically become resubscribed.
+
+![]({% image_buster /assets/img/email_settings/resubscribe_users.png %}){: style="max-width:90%;" }
+
+## Subscription pages and footers
+
+{% tabs %}
```
Comment (KellieHawks) [inline _docs/_user_guide/administrative/app_settings/manage_app_group/email_settings.md:131]:
```suggestion
{% tabs local %}
```

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! Two small fixes so both sets of tabs show

File: _includes/campaign_analytics.md [hunk 3]
```diff
@@ -100,11 +100,13 @@ If you want to simplify your view, click <i class="fas fa-plus"></i> **Add/Remov
 
 {% if include.channel == "email" %}
 
-#### Heat maps
+#### Heatmaps
 
-Additionally, you can see how successful different links in a single email campaign are using heat maps. Expand the **Total Clicks** dropdown and click **View Heat Map** to bring up a visual view of your email that shows the overall frequency and location of clicks within the lifespan of the campaign.
+Using heatmaps, you can see how successful different links in a single email campaign. From the **Campaign Analytics** section, go to the **Email Performance** panel. Click **Preview & Heatmap** to view a preview of your email campaign and the heatmap.
```
Comment (gsinghAppboy) [inline _includes/campaign_analytics.md:105]: Update 'campaign analytics' to Message Analytics (inclusive of Campaign or Canvas)

Comment (gsinghAppboy) [reply _includes/campaign_analytics.md:105]: Additional sentence at the end:

Alternatively, you can click the hyperlink found in the variant name

File: _includes/campaign_analytics.md [hunk 4]
```diff
@@ -100,11 +100,13 @@ If you want to simplify your view, click <i class="fas fa-plus"></i> **Add/Remov
...
 
-![email_analytics]({% image_buster /assets/img_archive/email_click_results_heatmap.gif %})
+In this view, you can use the **Show Heatmap** toggle to bring up a visual view of your email that shows the overall frequency and location of clicks within the lifespan of the campaign. In the **Link Table by Total Clicks** panel, you can view all of the links in your email campaign and sort by total clicks. This can provide additional insight on where your users navigate. 
+
```
Comment (gsinghAppboy) [inline _includes/campaign_analytics.md:108]: Add: "Lastly, you can click the download button to save the heatmap as an image."

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**